api/src/main/resources/leonardo.yaml

```diff
@@ -2481,6 +2481,9 @@ components:
         descriptorPath:
           type: string
           description: URL path to an app descriptor file. Only used if appType is CUSTOM.
+        workspaceId:
+          type: string
+          description: The workspace ID of the workspace in which to create the app.
```

Comment:
please be explicit that this is the Firecloud UUID, or "Terra DB ID" or similar.  We have very many references to "Workspace ID" which refer to something else

Reply:
figured it's in `leonardo.yaml`, so just copied over what leo has...but I guess updating this to more AoU specific makes sense too